circe/src/test/scala/org/http4s/circe/CirceSuite.scala

```diff
@@ -77,9 +77,7 @@ class CirceSuite extends JawnDecodeSupportSuite[Json] with Http4sLawSuite {
 
   test("json encoder should have json content type") {
     val maybeHeaderT: Option[`Content-Type`] = jsonEncoder[IO].headers.get(`Content-Type`)
-    assertEquals(
-      maybeHeaderT,
-      Some(`Content-Type`(MediaType.application.json)))
+    assertEquals(maybeHeaderT, Some(`Content-Type`(MediaType.application.json)))
```

Comment:
Did scalafmt want these?  I don't see the difference.

Reply:
Nah it works around inference issues. Would be solved with the header v2s aswell just wanted to make some progress